### ISSUE TITLE
After rewinding NeoGeo game inputs do not work

### ISSUE THREAD

**Issue by bopandrade** [libretro/fbalpha#146]
After rewinding a NeoGeo game the controllers are dead. I can still use functions such as load state/save state (not game related inputs, basically)

Loading state to a working point will bring controls back.
Loading state to a not working point will kill controls.

It sometimes rewinds without an issue but in most cases the inputs will be dead.

**Comment by fr500** [comment]
are you sure you're on an up to date core?

**Comment by fr500** [comment]
I personally fixed this a while ago and I tested thoroughly, if you are using something older or fba2012 it's irrelevant to this codebase.

https://github.com/libretro/fbalpha/commit/5f7f2258214300c886fa422497be3a11ea0c462e

I just played garou, the first 4 fights on android rewinding a lot and never had any issues.
If it's newer than that commit then try to be more specific. Give me steps to reproduce the issue including the game, the core commit and the retroarch version
